File: src/main/Java/boj_step3/boj2438.java
import java.io.*;

public class boj2438 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int n = Integer.parseInt(reader.readLine());
        int i = 1;
        while (i++ <= n) {
            int j = 1;
            while (j++ < i) writer.append("*");
            writer.newLine();
        }
        writer.close();
    }
}

File: src/main/Kotlin/others/boj2438.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val n = r.readLine()!!.toInt()
    for (i in 0 until n) {
        for (j in 0..i) w.write("*")
        w.write("\n")
    }
    w.close()
}